src/partest/scala/tools/partest/ReplTest.scala

```diff
@@ -65,17 +68,41 @@ object Hashless {
 /** Strip dynamic parts of LambdaMetafactory synthetic class names. */
 trait Lambdaless extends ReplTest {
   import Lambdaless._
-  override def normalize(s: String) = {
-    stripLambdaClassName(super.normalize(s))
-  }
+  override def normalize(s: String) = stripLambdaClassName(super.normalize(s))
 }
 object Lambdaless {
   private val lambdaless = """\$Lambda\$\d+/(?:0x[a-f0-9]{16}|\d+)(@[a-fA-F0-9]+)?""".r
   private def stripLambdaClassName(s: String): String = lambdaless.replaceAllIn(s, Regex.quoteReplacement("<function>"))
 }
 
+/** Normalize a REPL stack trace by stripping line numbers and count of elided frames. */
```

Comment:
it obfuscates `<console>` frames only, because details of REPL wrapping are fickle.